### ISSUE TITLE
Game crashes upon attempting to start.

### ISSUE THREAD

**Issue by rglx** [Sollace/MAtmos#18]
Installed 1.12.2 release of mod from the releases page, after using MultiMC to download and install LiteLoader, then Forge. 

Client log: https://paste.ee/p/KFba7

Reproducible if no other mods are present, without Forge, or with Forge and my regular complement of other mods
Will test to see if it reproduces on the Vanilla launcher.

**Comment by rglx** [comment]
Yup, exact same error on vanilla launcher, tested with LiteLoader+Forge, LiteLoader alone, and LiteLoader+Forge+other mods.

**Comment by DrakoAlcarus** [comment]
How did you manage to get a 1.12.2 release? I can only find a 1.10 version for download.

**Comment by DrakoAlcarus** [comment]
Got the same thing, not to mention Liteloader hates JEI. I personally loved matmos much better than Dynamic Surroundings. it's just not the same. Sucks to see Liteloader and Matmos get forgotten.

**Comment by ghost** [comment]
Same issue here, using vanilla launcher to install Forge and then LiteLoader. Those two install without problems but the game crashes on startup when MAtmos is in the mods folder.

Client log: https://paste.ee/p/iuOTJ
